### ISSUE TITLE
Verbose logging broken

### ISSUE THREAD

**Issue by ivasic** [3YOURMIND/django-migration-linter#86]
From the README:
```
--verbose or -v | Print more information during execution.
```

However, neither `-v` nor `--verbose` arguments to the `lintmigrations` command don't seem to work. Specifically:

``` bash
$ python src/manage.py lintmigrations -v
manage.py lintmigrations: error: argument -v/--verbosity: expected one argument
```

``` bash
$ python src/manage.py lintmigrations --verbose
manage.py lintmigrations: error: unrecognized arguments: --verbose
```

django-migration-linter==1.4.0
django==2.2.7
python 3.7.5

**Comment by David-Wobrock** [comment]
Hi @ivasic, thanks for reporting your issue.

I just checked and the verbosity is working as expected.
As the error message is stating: `argument -v/--verbosity: expected one argument` and the usage meesage also indicates `[-v {0,1,2,3}]`

Same for `--verbose`, it does not work because the argument is `--verbosity` (or its shortcut `-v`)

You can try `python src/manage.py lintmigrations -v 2`

Closing the issue

**Comment by ivasic** [comment]
Thanks David, that solves it indeed. However, you may want to update the README file since it incorrectly states `verbose` instead of `verbosity` and also doesn't seem to indicate that the argument is required

**Comment by David-Wobrock** [comment]
Good call, I corrected the readme :)
